lib/slas.js

```diff
@@ -120,7 +120,7 @@ const slas = {
                     handleCLIError('Could not get tenant: ', e.message, asJson)
                 }
             },
-            list: async (shortcode, tenantId, asJson) => {
+            list: async (tenantId, shortcode, asJson) => {
```

Comment:
order from cli expects the reverse:
`await slas.cli.client.list(options.tenant, options.shortcode, asJson);`